java/client/src/main/java/glide/api/models/commands/BitcountOptions.java

```diff
@@ -0,0 +1,29 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands;
+
+import glide.api.commands.BitmapBaseCommands;
+import lombok.RequiredArgsConstructor;
+
+/**
+ * Optional arguments for {@link BitmapBaseCommands#bitcount(String, long, long, BitcountOptions)}.
+ *
+ * @see <a href="https://redis.io/commands/bitcount/">redis.io</a>
+ */
+@RequiredArgsConstructor
+public enum BitcountOptions {
```

Comment:
Rename to ~`BitCoinOptions`~  `BitCountOptions`

Reply:
I think this should remain as `BitcountOptions` as the command is actually `Bitcount`.

Reply:
It is used in `bitpos` too. Need to rename according to the common sense.